python/spec/dependabot/python/update_checker/pipenv_version_resolver_spec.rb

```diff
@@ -122,33 +122,6 @@
       end
     end
```

Comment:
Looks like the above error message is `CRITICAL:pipenv.patched.notpip._internal.index.package_finder:Could not find a version that satisfies the requirement pytest==10.4.0`